src/app/client/src/app/modules/public/module/signup/services/signup/signup.service.ts

```diff
@@ -20,6 +20,14 @@ export class SignupService {
     return this.learnerService.post(options);
   }
 
+  generateOTPforAnonymousUser(data, captchaResponse) {
+    const options = {
+      url: this.configService.urlConFig.URLS.OTP.ANONYMOUS.GENERATE + '?captchaResponse=' + captchaResponse,
```

Comment:
Why are we using new route for anonymous. Generate OTP should be common for any use-case.